magento1-module/app/code/local/Divante/VueStorefrontBridge/controllers/RewriteController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+require_once Mage::getModuleDir('controllers', 'Divante_VueStorefrontBridge') . DS . 'AbstractController.php';
+class Divante_VueStorefrontBridge_RewriteController extends Divante_VueStorefrontBridge_AbstractController
+{
+    public function targetAction()
+    {
+        if (!$this->_checkHttpMethod('GET')) {
+            return $this->_result(405, 'Method not allowed');
+        }
+
+        $requestPath = $this->getRequest()->getParam('request_path');
+        if ($requestPath) {
+            $reader = Mage::getSingleton('core/resource')->getConnection('core_read');
+
+            $select = $reader->select()
+                             ->from('core_url_rewrite', ['target_path'])
+                             ->where('request_path = ?', $requestPath)
+                             ->where('store_id IN (?)', [Mage_Core_Model_App::ADMIN_STORE_ID, (int)Mage::app()->getStore()->getId()])
```

Comment:
I've always tried to stay away from using the store id in the API, I've had issues with false matches on that.
might we require it to be set in the API call?